_safari/Save to Pocket/Base.lproj/Main.storyboard

```diff
@@ -107,12 +96,12 @@
                                 <imageCell key="cell" refusesFirstResponder="YES" alignment="left" imageScaling="proportionallyDown" image="bgTriangle" id="XZb-he-vmh"/>
                             </imageView>
                             <textField verticalHuggingPriority="750" horizontalCompressionResistancePriority="250" translatesAutoresizingMaskIntoConstraints="NO" id="X1M-w3-9da">
-                                <rect key="frame" x="35" y="134" width="199" height="132"/>
+                                <rect key="frame" x="35" y="182" width="199" height="84"/>
                                 <constraints>
                                     <constraint firstAttribute="width" constant="195" id="RrD-zd-YNx"/>
                                 </constraints>
-                                <textFieldCell key="cell" selectable="YES" id="vuh-O7-RhV">
-                                    <font key="font" size="18" name="Doyle-Medium"/>
+                                <textFieldCell key="cell" id="vuh-O7-RhV">
+                                    <font key="font" metaFont="system" size="18"/>
```

Comment:
Does this mean we are dropping Doyl as the font? Or does it get set up as the system some other way?

Reply:
That looks like a mistake. Let me peek.